widgetry/src/canvas.rs

```diff
@@ -161,6 +161,10 @@ impl Canvas {
                     );
                 }
             }
+
+            if input.left_mouse_double_clicked() {
+                self.zoom(8.0, self.map_to_screen(map_pt));
```

Comment:
The zoom change feels too low to me -- how do you feel about doubling to 16? Then it's about 2 or 3 clicks to go from max unzoomed to seeing the zoomed in view.

Reply:
To me 16 feels too disorienting -  especially if you're on a smaller map. e.g. this video starts zoomed all the way out on montlake:

https://user-images.githubusercontent.com/217057/130325729-3af0bddd-7fd9-430d-ae9c-39638530f0ea.mov

To help maintain context, it'd help if the zoom were animated, but it seems like a nontrivial addition.

The current zoom step of 8 is already more aggressive than apple/google/osm maps. Having messed around with it a bit, I prefer it as is.

Reply:
Sounds good, let's keep it.

> it'd help if the zoom were animated, but it seems like a nontrivial addition.

We do have https://github.com/a-b-street/abstreet/blob/71b54b3ed709952952dd22df20075c0768c51dd7/game/src/common/warp.rs#L24, but the effect is pretty disorienting. And it's very unclear how to bake this in at the widgetry Canvas level. So agreed, nontrivial.